ruff.schema.json

```diff
@@ -1415,7 +1405,7 @@
             "null"
           ],
           "items": {
-            "type": "string"
+            "$ref": "#/$defs/string"
```

Comment:
These changes looks weird, but they refer to `Name` struct, which means String in schemars. That is why they use `defs/string` instead of `string`.

https://github.com/astral-sh/ruff/blob/c4240076459bc9128fee6f83cbf0f51d134b663b/crates/ruff_workspace/src/options.rs#L1969-L1976

https://github.com/astral-sh/ruff/blob/c4240076459bc9128fee6f83cbf0f51d134b663b/crates/ruff_workspace/src/options.rs#L1959-L1967

https://github.com/astral-sh/ruff/blob/c4240076459bc9128fee6f83cbf0f51d134b663b/crates/ruff_python_ast/src/name.rs#L204-L227